vulnerabilities/tests/test_curl.py

```diff
@@ -0,0 +1,52 @@
+#
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# VulnerableCode is a trademark of nexB Inc.
+# SPDX-License-Identifier: Apache-2.0
+# See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+# See https://github.com/nexB/vulnerablecode for support or download.
+# See https://aboutcode.org for more information about nexB OSS projects.
+#
+
+import json
+import os
+from unittest.mock import patch
+
+import requests
+from bs4 import BeautifulSoup
+# from importer import AdvisoryData
+from univers.versions import SemverVersion
+from vulnerabilities.importers import curl
+from vulnerabilities.tests import util_tests
+
+BASE_DIR = os.path.dirname(os.path.abspath(__file__))
+TEST_DATA = os.path.join(BASE_DIR, "test_data/curl")
+
+
+def test_is_semver_version():
```

Comment:
Fetching the URL inside the test is a bad idea. 
it makes tests depend on internet connectivity.
try to use a static file instead and please add a test to the importer 
ex:
https://github.com/nexB/vulnerablecode/pull/897/files#diff-1c1d12198c8d593ef053851512de82d7ce743fa976dfe870777ca9032db08606R21-R36